src/Microsoft.Extensions.DependencyInjection.Specification.Tests/DependencyInjectionSpecificationTests.cs

```diff
@@ -158,6 +158,33 @@ public void MultipleServiceCanBeIEnumerableResolved()
         }
 
         [Fact]
+        public void RegistrationOrderIsPreservedWhenServicesAreIEnumerableResolved()
+        {
+            // Arrange
+            var collection = new ServiceCollection();
+            collection.AddTransient(typeof(IFakeMultipleService), typeof(FakeOneMultipleService));
+            collection.AddTransient(typeof(IFakeMultipleService), typeof(FakeTwoMultipleService));
+
+            var provider = CreateServiceProvider(collection);
+
+            collection.Reverse();
```

Comment:
Filed https://github.com/aspnet/DependencyInjection/issues/419 to rename the one in the test specification.